scylla.go

```diff
@@ -4,17 +4,96 @@ import (
 	"fmt"
 	"math"
 	"strconv"
+	"strings"
 	"sync/atomic"
 )
 
 // scyllaSupported represents Scylla connection options as sent in SUPPORTED
 // frame.
+// FIXME: Should also follow `cqlProtocolExtension` interface.
```

Comment:
What would it bring to have the `scyllaSupported` struct also implementing the `cqlProtocolExtension` interface? When would this be used?

Reply:
It should not be hard to do, but I don't want to include it there since it's completely unrelated to the main cause of this patch (and it's not even anywhere on my priority list at the moment). And it for sure will trigger a lengthy discussion, which will cause unneeded delay for this series to be merged.